webrtc/RTCPeerConnection-mandatory-getStats.https.html

```diff
@@ -23,18 +23,18 @@
     "packetsReceived",
     "packetsLost",
     "jitter",
-    "packetsDiscarded",
     "framesDropped"
   ],
   RTCInboundRtpStreamStats: [
```

Comment:
Not a WPT question, but this does make me wonder if inbound-rtp.mid and outbound-rtp.mid should be "mandatory to implement", but that ship may have sailed already